tensorflow/tools/api/golden/v1/tensorflow.tpu.pbtxt

```diff
@@ -18,7 +18,7 @@ tf_module {
   }
   member_method {
     name: "bfloat16_scope"
-    argspec: "args=[], varargs=None, keywords=None, defaults=None"
+    argspec: "args=[\'name\'], varargs=None, keywords=None, defaults=[\'\'], "
```

Comment:
Does the golden need to be regenerated?

Reply:
yes, I have updated the golden. 
thanks!